prov/rxm/src/rxm.h

```diff
@@ -214,9 +214,8 @@ struct rxm_rma_iov_storage {
 enum rxm_buf_pool_type {
 	RXM_BUF_POOL_RX		= 0,
 	RXM_BUF_POOL_START	= RXM_BUF_POOL_RX,
-	RXM_BUF_POOL_TX_MSG,
-	RXM_BUF_POOL_TX_START	= RXM_BUF_POOL_TX_MSG,
-	RXM_BUF_POOL_TX_TAGGED,
+	RXM_BUF_POOL_TX,
+	RXM_BUF_POOL_TX_START	= RXM_BUF_POOL_TX,
```

Comment:
Can you please add some details to the commit message regarding why separate pools are not needed anymore?

Reply:
The details were added. Thanks!